From 199144e9509dd91100661ae0c948135d23221ed6 Mon Sep 17 00:00:00 2001
From: Hien Huynh <hien.huynh.px@renesas.com>
Date: Wed, 4 Mar 2020 16:44:55 +0700
Subject: [PATCH 186/197] drm: rcar-du: workaround to display DPAD0 and LVDS-IF
 at the same time in r8a774c0 SoC.

In R8A774C0 SoC, there are 3 outputs of DU:
 0 - DPAD (DU0 or DU1)
 1 - LVDS0 (DU0)
 2 - LVDS1 (DU1)

With RZ/G2E board, route of connected encodeders is:
 - DPAD0 -> HDMI
 - LVDS0(even data) + LVDS1(odd data) -> Dual channels LVDS.

To display multiple output at the same time, this patch hard fixed for
DPAD0 set with DU1.

Signed-off-by: Hien Huynh <hien.huynh.px@renesas.com>
---
 drivers/gpu/drm/rcar-du/rcar_du_drv.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/rcar-du/rcar_du_drv.c b/drivers/gpu/drm/rcar-du/rcar_du_drv.c
index 1429aff..302d040 100644
--- a/drivers/gpu/drm/rcar-du/rcar_du_drv.c
+++ b/drivers/gpu/drm/rcar-du/rcar_du_drv.c
@@ -149,7 +149,7 @@ static const struct rcar_du_device_info rcar_du_r8a774c0_info = {
 		 * R8A774C0 has one RGB output and two LVDS outputs
 		 */
 		[RCAR_DU_OUTPUT_DPAD0] = {
-			.possible_crtcs = BIT(0) | BIT(1),
+			.possible_crtcs = BIT(1),
 			.port = 0,
 		},
 		[RCAR_DU_OUTPUT_LVDS0] = {
-- 
2.7.4

